src/Microsoft.Health.Dicom.Core.UnitTests/Extensions/ExtendedQueryTagEntryExtensionsTests.cs

```diff
@@ -87,10 +96,19 @@ public void GivenStandardTagAsKeyword_WhenNormalizing_ThenVRShouldBeFilled()
             Assert.Equal(normalized.Path, expectedPath);
         }
 
+        [Fact]
+
+        public void GivenInvalidTagWithoutVR_WhenNormalizing_ThenShouldNotThrowException()
+        {
+            // track bug  https://microsofthealth.visualstudio.com/Health/_workitems/edit/81015
```

Comment:
Won't this comment become moot as soon as you check this in because you're fixing it? Probably best not to leave the URL in here